android/build.gradle

```diff
@@ -17,6 +16,7 @@ rootProject.allprojects {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://maven.google.com' }
```

Comment:
Why do we need this?

Reply:
It was written in the docs: https://segment.com/docs/connections/destinations/catalog/firebase/ 
Im not sure what it does or if it is even neccesary. Im kind of noob on this.

Reply:
I spoke with my team on this. We found out that these are required in the project level gradle, so this is not neccesary here. I will remove it.